source/diagrams/manage-diagrams/create-diagrams.txt

```diff
@@ -0,0 +1,30 @@
+.. _rm-create-diagrams:
+
+===============
+Create Diagrams
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can create new diagrams in the Mapping tab of Relational Migrator.
```

Comment:
We should also specify here that you can create multiple diagrams and can select between them at any one time and a slight description of why you would want to do that (viewing specific entities of the data model). Not sure if there is a maximum number of diagrams but it might be good to specify in TR.

Reply:
Done!

Reply:
Confirmed with Lavender that there is no maximum number of diagrams!